ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx

```diff
@@ -185,6 +212,19 @@ const RunComponent = () => {
                 </TableCell>
               </TableRow>
             )}
+            <TableRow>
+              <TableCell>Logs</TableCell>
+              <TableCell>
+                <Link onClick={() => handleDownload(ortRun.id)}>
+                  <Button
+                    variant='outline'
+                    className='font-semibold text-blue-400'
+                  >
+                    Download ZIP file
```

Comment:
IMO we should say *what* is being downloaded (and the fact that the archive is in ZIP format is less important), so I propose "Download log archive".